scripts/mkcloud

```diff
@@ -526,7 +543,7 @@ function setupadmin()
     fi
     echo "you can now proceed with installing crowbar"
     # prevent jumbo frames from going out
-    if [[ $want_mtu_size ]] ; then
+    if [[ $want_mtu_size -gt 1500 ]] ; then
```

Comment:
btw is the iptables code correct? Why do you set it to 1500? This is the MTU value but the MSS should be at least 40 bytes less than that right?